src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/CounterGroup.cs

```diff
@@ -171,7 +174,7 @@ private void EnableTimer(float pollingIntervalInSeconds)
                         IsBackground = true,
                         Name = ".NET Counter Poller"
                     };
-                    s_pollingThread.InternalUnsafeStart();
+                    s_pollingThread.Start();
```

Comment:
Is this change intended?

Reply:
Yes, `InternalUnsafeStart` was there only for browser and only for this use-case, that no longer works anyway.